SafeguardSessions/README.md

```diff
@@ -60,7 +62,15 @@ connection was closed: Could not establish trust relationship for the SSL/TLS
 secure channel."
 ```
 
-You might have forgotten to import the CA X.509 certificate of your SPS appliance described in the [Installation and usage](#installation-and-usage) section.
+You might have forgotten to import the CA X.509 certificate of your SPS appliance described in the [Installation and usage] section.
+
+---
+
+If you have successfully upgraded your custom connector and tried to fetch data, but receive the following message, revisit the [Upgrading the Power BI Connector] section.
+
+```
+Your version of the connector (ConnectorVersion) is not compatible with your SPS version (SPSVersion). For a connector version that is compatible with your SPS version, visit the official release page of the connector: https://github.com/OneIdentity/SafeguardPowerBI/releases
```

Comment:
One minor nit-pick, I just recognized. If we read the README.md on github (not in raw), we can only read the error message by scrolling right with the slider. I think its a bad user experience in general, so we could break the error message in multiple lines for better readability.

Reply:
I tried 2 versions. The second one is the winner.

As the view is responsive, scrolling is needed if the page width is too small. But for a desktop browser and FHD resolution, the current separation is good enough.